packages/lab/src/Table/stories/TableColumnRenderers.stories.mdx

```diff
@@ -0,0 +1,202 @@
+import { Meta, Story, Preview } from "@storybook/addon-docs";
+
+import LinkTo from "@storybook/addon-links/react";
+
+import { Usage } from "storybook-root/blocks/Usage";
+
+import { HvTable } from "../..";
+
+import { 
+  AllColumnRenderers, 
+  TextColumnRenderer, 
+  NumberColumnRenderer, 
+  DateColumnRenderer,
+  ExpandColumnRenderer,
+  SwitchColumnRenderer,
+  TagColumnRenderer,
+  ProgressColumnRenderer,
+  DropdownColumnRenderer,
+} from "./TableColumnRenderers.stories.js";
+
+<Meta title="Lab/Table Column Renderers" />
+
+# Table column renderers
+
+The UI Kit library provides a collection of functions that work as **renderers** that ease the generation of various common column configurations.
```

Comment:
I would add the following sentence here. 

They add a set of out of the box features, not meant to be 100% feature complete, but instead, ease the majority of use-cases we have been encountering. If you need any customization or extension to these rendeders, please feel free to copy and customize them.